umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md

```diff
@@ -92,41 +100,34 @@ For any other microservice, follow these steps to enable the LoadBalancer servic
    ```
 
    Replace `<external-ip>` with the external IP address of the node.
-4. Scroll to the `spec.type` section and change the value from ClusterIP to
-   LoadBalancer.
-5. Click **Save** to apply the changes.
+4. Go to the `spec.type` section and change the value from `ClusterIP` to
+   `LoadBalancer`. 
+5. After saving, your changes will be applied automatically and the service will 
+   be updated. Now, you can access the service with the configured address.
 
-If you installed the United Manufacturing Hub on your local machine, either
-using the Management Console or the command line, you also need to map the port
-exposed by the k3d cluster to a port on your local machine. To do that, run the
-following command:
+### Port forwarding
 
-```bash
-k3d cluster edit {{< resource type="cluster" name="name" >}} --port-add "<local-port>:<cluster-port>@server:0"
-```
-
-Replace `<local-port>` with a free port number on your local machine, and
-`<cluster-port>` with the port number of the service.
+If you don't want to create a LoadBalancer service, effectively exposing the
+microservice to anyone that has access to the host IP address, you can forward 
+the port to your local machine.
 
-### Port forwarding in {{% resource type="lens" name="name" %}}
+1. Execute the following command to get the list of available services and remember 
+   the service to be exposed.
```

Comment:
"Execute the following command to list the services and note the name of the one you want to port-forward and the internal port that it uses"

Reply:
updated